lib/src/sv_common.c

```diff
@@ -158,18 +158,18 @@ sign_or_verify_data_free(sign_or_verify_data_t *self)
   free(self);
 }
 
-// Convert Unix timestamp (seconds since 1970) to 1601-based timestamp
-void
-convert_unix_to_1601(int64_t *timestamp)
+// Convert Axis Unix timestamp (microseconds since 1970) to 1601-based timestamp (100-ns intervals)
```

Comment:
This is not Axis specific. The Unix Epoch time can be interpreted in seconds, ms, us, ns depending on the number of digits. See for example https://www.epochconverter.com/

If you want to build a true converter you should analyze the timestamp to determine in what scale it represents the time.
I think it is fine here to assume microseconds, but maybe name the function `convert_unix_us_to_1601(...)`